dali/kernels/reduce/reduce_drop_dims.h

```diff
@@ -241,18 +279,23 @@ struct DropDims {
     // Warning: intentional fall-through!
     #define REINDEX_CASE(idx)\
       case 2*idx:\
-        out += index / div[idx] * mul[idx];\
+        if (idx < kMaxDims)\
+          out += static_cast<uint64_t>(index) / div(idx) * mul[cuda_min(idx, kMaxDims-1)];\
```

Comment:
How about creating `mul` function as well to have a consistent notation?
Idea, not a must.

Reply:
actually, if I used mul as a function here (and mod function three lines below) I could get away without this silly `min(idx, kMaxDims-1)` which unnecessary, but clang is just not smart enough to silence the warning.